.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - directory: /
+    open-pull-requests-limit: 5
+    package-ecosystem: gomod
+    schedule:
+      interval: weekly
+
+  - directory: docs
+    open-pull-requests-limit: 5
+    package-ecosystem: npm
+    schedule:
+      interval: weekly
```

Comment:
dunno about this, personally I'm not too hot on reviewing these updates, but maybe it's fine to just smoke check the generated website you get on PRs